src/components/MemoryItem.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+MemoryItem.propTypes = {
+  image: PropTypes.node,
+  subtitle: PropTypes.string,
```

Comment:
Maybe add isRequired? Be as explicit and strict as possible to make your life easier while developing. This wouldn't throw an error if no image & subtitle is provided
```suggestion
  image: PropTypes.node.isRequired,
  subtitle: PropTypes.string.isRequired,
```

Reply:
Maybe later :)